.env

```diff
@@ -86,6 +86,7 @@ _APP_MAINTENANCE_RETENTION_EXECUTION=1209600
 _APP_MAINTENANCE_RETENTION_ABUSE=86400
 _APP_MAINTENANCE_RETENTION_AUDIT=1209600
 _APP_USAGE_AGGREGATION_INTERVAL=30
```

Comment:
Let's make sure all env prefixes are consistent. According to our last discussion, we should probably move to stats namespace.